cli/js/web/console.ts

```diff
@@ -684,6 +684,8 @@ const countMap = new Map<string, number>();
 const timerMap = new Map<string, number>();
 const isConsoleInstance = Symbol("isConsoleInstance");
 
+const builtinConsole = globalThis.console;
```

Comment:
This effectively means you get back instance of `Console` back

Reply:
It is captured into `Console` class, running before `bootstrapMainRuntime`/`bootstrapWorkerRuntime` where `globalThis.console` is bound to `Console` class.

Reply:
Not sure I understand, could you elaborate?

Reply:
(Sorry, was interrupted in reply yesterday)

The code grabs a reference to the v8's builtin `console` object (storing in `builtinConsole`), before `globalThis.console` is assigned the deno implementation of `console`.
Here it is done just before the `Console` class is defined, so it should work out in execution order.
I believe the code is run as part of snapshot process in build.rs, but it did work.

I assume this will also work out the same for "userland" workers.

The actual overwriting of `globalThis.console` to deno's, is done as part of ["boostrap.mainRuntime()"](https://github.com/denoland/deno/blob/master/cli/lib.rs#L158) and ["bootstrap.workerRuntime()"](https://github.com/denoland/deno/blob/master/cli/ops/worker_host.rs#L67), after the workers are initialized with the snapshot.

Reply:
Thanks for explanation. I think what we want to do is hook up `Deno.core.print` into inspector to pipe messages. (`Deno.core.print` is the mechanism that console uses to actually output message to stdout/stderr)

Reply:
There is a lot of additional data in the messages from v8 not readily available through the print function such as line numbers, stack traces with scriptId's and urls, executionContextId's (possibly used with multiworker debugging). Some stuff like manipulation of the js variables, such as marking for gc retention, I assume for follow-up drilling down into complex datastructures. They may not be ideal, but one gets alot for free.